CogniteSdk/src/Resources/DataPoints.cs

```diff
@@ -57,6 +58,27 @@ public async Task<EmptyResponse> CreateAsync(DataPointInsertionRequest points, C
             var req = Oryx.Cognite.DataPoints.create(points);
             return await RunAsync(req, token).ConfigureAwait(false);
         }
+        
+        /// <summary>
+        /// Create data points, applying Gzip compression at level <paramref name="compression"/>.
+        /// </summary>
+        /// <param name="points">Data Points to create</param>
+        /// <param name="compression">Compression level</param>
+        /// <param name="token">Optional cancellation token.</param>
+        /// <returns>Empty response</returns>
+        public async Task<EmptyResponse> CreateWithGzipAsync(
```

Comment:
Should we make this as an overload instead? I.e call it `CreateAsync` but with the same comment and arguments.

Reply:
Sure, with compression level last so that you don't need to add `null` to be able to specify a cancellationtoken?

Reply:
Never mind.